www/app/categories/categories.html

```diff
@@ -5,7 +5,7 @@
     <ul class="categories-list">
       <li class="item" ng-class="{ 'selected': isSelected(category)}" 
         ng-repeat="category in categories" 
-        ng-click="selectCategory(category)">
```

Comment:
big error!!!!

in the function showConfirmSubscribe you are assuming that category is selected, but you remove that function!